specs/WebResourceRequestedWorkers.md

```diff
@@ -0,0 +1,210 @@
+WebResourceRequested events for workers
+===
+
+# Background
+Currently [WebResourceRequested events](https://docs.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/iwebview2webview5?view=webview2-0.8.355#add_webresourcerequested) are raised only for top level document HTTP requests. We are extending [AddWebResourceRequestedFilter](https://docs.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/iwebview2webview5?view=webview2-0.8.355#addwebresourcerequestedfilter) API to allow you additionally to subscribe to events raised from Service or Shared workers and fixing an issue where out of process iframes did not raise WebResourceRequested events as part of the main page.
+
+# Examples
```

Comment:
No samples for Removing a filter, or any error handling.  Any interesting examples to provide for forgetting to set a filter?

Reply:
About forgetting to set a filter we have in [the doc for event subscription](https://docs.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/icorewebview2?view=webview2-1.0.1264.42#add_webresourcerequested): "At least one filter must be added for the event to run."